docker/docker-compose.yml

```diff
@@ -0,0 +1,30 @@
+concourse-db:
```

Comment:
👍  pinning versions for reproducibility.

Looks like this has been implemented now, but the comment wasn't collapsed.